cpu/saml21/Makefile.features

```diff
@@ -1 +1,6 @@
--include $(RIOTCPU)/sam0_common/Makefile.features
+# samr30 does not contain a TRNG
+ifeq (,$(findstring samr30,$(CPU_MODEL)))
```

Comment:
Please replace it with whitelisting boards and  a comment on which CPU_MODEL have the feature instead of the opposite.

Reply:
stm32 and kinetis use a `BOARDS_WITHOUT_HWRNG` blacklist - I think this is easier to maintain.
I've added one to saml21 too.

Reply:
Defining the `BOARDS_WITHOUT_HWRNG` variable without clearing it in `makefiles/info-global.inc.mk` is somehow also wrong even if harmless.

But as it is currently done for the other family I agree.